db/migrate/20151124183140_remove_talks.rb

```diff
@@ -1,5 +1,6 @@
 class RemoveTalks < ActiveRecord::Migration
   def change
-    drop_table :talks
+    drop_table :talks do
```

Comment:
@cawel I guess we can always learn something :)
Thanks for investigating.
I am AFK but if anyone can merge, please go ahead...